modules/ossm-rn-known-issues.adoc

```diff
@@ -46,6 +44,10 @@ Workaround: Use the `techPreview` `jwksResolverExtraRootCA` field to enable addi
 +
 After enabling the `spec.security.controlPlane.mtls` setting in the SMCP, the Kiali console displays the following error message `No subsets defined`.
 
+* https://issues.redhat.com/browse/OSSM-1505[OSSM-1505] `ServiceMeshExtension` failed to be ready. If you are using Service Mesh with `ServiceMeshExtension` on OpenShift Container Platform 4.11 then the `ServiceMeshExtension` resource never becomes ready. If you inspect the issue using `oc describe ServiceMeshExtension` you will see the following error:  `stderr: Error creating mount namespace before pivot: function not implemented`.
++
+Workaround: `ServiceMeshExtension` was deprecated in {SMProductShortName} 2.2. Migrate from `ServiceMeshExtension` to the `WasmPlugin` resource. For more information see Migrating from `ServiceMeshExtension` to `WasmPlugin` resources.
```

Comment:
Is it possible to link to the actual migration page?

Reply:
@jwendell unfortunately there are some limitations and associated practices relating to the existing portal, and providing that link in the content is not possible. There is a long-term project underway to migrate to a different portal. When that migration happens, adding such links will be possible.

Reply:
That's sad :/